src/main/controller/StrengthCheckerUIControl.java

```diff
@@ -0,0 +1,30 @@
+package controller;
+import useCases.ComputePasswordStrength;
+
+public class StrengthCheckerUIControl extends UIController {
+    String result;
+    String suggestion;
+
+    public String getResult() {
+        return result;
+    }
+
+    public void setResult(String result) {
+        this.result = result;
+    }
+
+    public String getSuggestion() {
+        return suggestion;
+    }
+
+    public void setSuggestion(String suggestion) {
```

Comment:
Does this get called anywhere outside the class? If not, can it go away?

Reply:
@pritishmishra